src/app/store/accounts/blockchain/stacks/stacks-accounts.ts

```diff
@@ -72,7 +72,7 @@ const selectStacksWalletState = createSelector(
 
 const softwareAccountsState = atom<SoftwareStacksAccount[] | undefined>(get => {
   const store = get(storeAtom);
-  const addressVersion = get(addressNetworkVersionState);
+  const addressVersion = get(addressNetworkVersionState) || AddressVersion.TestnetSingleSig;
```

Comment:
Don't really know what we want to do here
Maybe get the AddressVersion based on the subnet network parent id?

Reply:
Sorry for slow reply, was off last week. 

What's the idea here? That because there are new chain ids we want to default to testnet single sig (as atom would otherwise be undefined). How can we infer which is the correct `AddressVersion` to be used for a subnet?

Reply:
Thanks for your answer @kyranjamie 
I'll close this PR in favor of https://github.com/hirosystems/wallet/pull/4008